openstack/tools/func_test_tools/common.sh

```diff
@@ -44,3 +44,31 @@ apply_func_test_pr ()
         -f './src/test-requirements*.txt' \
         "$msg"
 }
+
+allocate_port ()
+{
+    # Returns address of port created.
+    #
+    local net_name=$1
+    local port_name=$2
+    openstack port create --network $net_name $port_name -c fixed-address -f value
+}
+
+create_zaza_vip ()
+{
+    # Allocates a vip ensuring to use existing ones if they exist.
+    #
+    # Returns the address of the vip.
+    #
+    local vip_id=$1
+    # We use the same naming convention as ../tools/allocate_vips.sh to avoid conflicts and re-use
+    # those vips.
+    vip_port_name=ps6-vip-ip$((vip_id-1))
```

Comment:
Is this vip_id-1 correct? I think the caller passes (1, 0)

> for ((i=2;i;i-=1)); do echo $((i-1)); done
1
0

Reply:
actually the caller passes $i but would make more sense if it passed $i-1 so ill fix it